build/reactable.js

```diff
@@ -443,6 +443,10 @@ window.ReactDOM["default"] = window.ReactDOM;
                     onClick: this.handleClick.bind(this)
                 };
 
+                if (typeof this.props.style !== 'undefined') {
```

Comment:
I've used typeof as `typeof(this.props.style)` elsewhere in the codebase, let's keep that consistent